.github/workflows/ci.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```

Comment:
@henryiii should we specify the python version, or should we lock this to `v3.0.0` and wait for the maintainers to make it optional again?

Reply:
I think we should start specifying `python-version: 3.x` and avoid getting locked to `setup-python/v3`. I'm hoping that https://github.com/actions/setup-python/issues/421 would get resolved by restoring the default behavior, but nothing has happened there yet.

Reply:
```suggestion
      with:
        python-version: 3.x
```

Reply:
Thanks!